core/src/components/back-button/back-button.tsx

```diff
@@ -8,6 +8,10 @@ import { createColorClasses, hostContext, openURL } from '../../utils/theme';
 
 /**
  * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
+ *
+ * @part native - The native HTML button element that is rendered around the content.
```

Comment:
```suggestion
 * @part native - The native HTML button element that is rendered around the back button text.
```

or something similar - saying "the content" is a little vague and makes me think of ion-content